packages/otel/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "3.0.0",
   "type": "module",
   "engines": {
-    "node": ">=20.6.1"
+    "node": ">=18"
```

Comment:
This was previously `"^18.14.0 || >=20"`, we might potentially also choose `"^18.14.0 || >=20.6.1"`